llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -201,9 +204,17 @@ def save_model_patch(*args: Any, **kwargs: Any):
     if mlflow_logger.model_registry_uri is not None:
         mlflow.set_registry_uri(mlflow_logger.model_registry_uri)
 
+    if is_peft:
+        transformers_in_memory_model = {
+            'model': AutoModel.from_pretrained(transformers_model),
```

Comment:
why not just pass the model in directly to this function since its already in memory?

Reply:
Good catch, fixed